public/js/post/post.js

```diff
@@ -22,15 +39,121 @@ var CourseCode = React.createClass({
         }
 
         return (
-            <div id={this.props.courseID} className="course">
-                <p className={categoryClasses} onClick={this.toggleFullInfo}>  {this.props.courseID.toUpperCase() + "H"}  </p>
-                <div id={this.props.courseID + '_info'} className='more-info'>
-                    <p className={courseClasses}> {getCourseTitle(this.props.courseID)} </p>
+            <div id ={this.props.courseIDs[0]} className='course'>
+                <p className={categoryClasses} onClick={this.toggleFullInfo}> {this.getCategoryName()} </p>
+                <div id= {this.props.courseIDs[0] + '_info'} className='more-info'>
+                    {this.props.courseIDs.map(function (course) {
+                         return <p className={courseClasses}> {getCourseTitle(course)} </p>
+                    })}
                 </div>
             </div>
         );
     }
 })
 
+var MultipleCourseCode = React.createClass({
+    getInitialState: function() {
+        return {
+            completed: false,
+            completedTextBoxes: 0
+        }
+    },
+    
+    toggleFullInfo: function() {
+          $('#' + this.props.courseID + ' > .more-info').toggle();
+    },
+
+    checkIfCompleted: function() {
+       this.setState({completed: this.state.completedTextBoxes === this.props.data.textBoxNumber});
+    },
+
+    handleKeyDown: function(e) {
+        if (e.keyCode === 13) {
+            if (this.state.completedTextBoxes < this.props.data.textBoxNumber) {
+                 this.setState({completedTextBoxes: this.state.completedTextBoxes +=1})
+            }
+
+            this.checkIfCompleted();
+        }  
+    },
+
+    render: function() {
+
+        var categoryClasses = 'code';
+        var courseClasses = 'full_name';
+
+        if (this.state.completed) {
```

Comment:
Can you change the CSS selector so that you can just make the outer div have a class of `selected` rather than apply it to both inner parts?